### ISSUE TITLE
type != "fullslide" should exclude titles, subtitles and captions

### ISSUE THREAD

**Issue by wfmackey** [MattCowgill/grattantheme#20]
@MattCowgill  What's the benefit of non-fullslide plots being saved with titles, subtitles and captions? They're only going to be used in reports or pasted into ppts; and both require chart-only formats. (Or am I missing something?)

So would it be better to remove titles, subtitles and captions from the plot before a save when `type != "fullslide"`?

**Comment by wfmackey** [comment]
I guess the only thing to gain is, for a plot:
```
p + 
  labs(title = "Old trees are big",
          subtitle = "Tree size by age",
          x = "Age",
          y = "",
          caption = "Notes: these are orange trees. Source: ships with R.")
```
a lot of the time you'll want to save a report/insert-into-ppt slide **and** a fullslide to post on the internet. Without proper save defaults you have to create the plot without labels, `grattan_save("", type = "normal")` then add labels, then `grattan_save("", type = "fullslide").

**Comment by wfmackey** [comment]
ie better practice to create a single ggplot object and save it in various ways than have to create and update plots depending on the save type?

**Comment by wfmackey** [comment]
Oop - sorry, I just saw this. Came here to merge a pull request that updates it. Happy for you to overwrite with yours.

**Comment by MattCowgill** [comment]
OK, no worries. Let's go ahead with the pull request but I might make some further minor changes

**Comment by wfmackey** [comment]
I also note that you wrote that last comment at 4am and I am sorry for your sleep.

**Comment by MattCowgill** [comment]
4:23, it's fine

**Comment by wfmackey** [comment]
Fixed in #24
